### ISSUE TITLE
LwjglAWTCanvas limitations

### ISSUE THREAD

**Issue by metc** [hevs-isi/gdx2d#87]
The `DemoJuliaFractal` cannot be run from the demo selector.
The demo runs fine using the `DemoJuliaFractal.main` method.
 Here is the full stack trace error:
```
java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:39)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:27)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:513)
	at hevs.gdx2d.lib.gui.DemoSelectorGUI$TestList$RunButton$1.run(DemoSelectorGUI.java:221)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:209)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:672)
	at java.awt.EventQueue.access$400(EventQueue.java:81)
	at java.awt.EventQueue$2.run(EventQueue.java:633)
	at java.awt.EventQueue$2.run(EventQueue.java:631)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.AccessControlContext$1.doIntersectionPrivilege(AccessControlContext.java:87)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:642)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:269)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:184)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:174)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:169)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:161)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:122)
Caused by: java.lang.UnsatisfiedLinkError: com.badlogic.gdx.graphics.g2d.Gdx2DPixmap.setBlend(I)V
	at com.badlogic.gdx.graphics.g2d.Gdx2DPixmap.setBlend(Native Method)
	at com.badlogic.gdx.graphics.g2d.Gdx2DPixmap.<clinit>(Gdx2DPixmap.java:51)
	at com.badlogic.gdx.graphics.Pixmap.<init>(Pixmap.java:114)
	at hevs.gdx2d.demos.simple.DemoJuliaFractal.<init>(DemoJuliaFractal.java:38)
	... 19 more
```

Moreover, the framerate is *not* limited with `LwjglAWTCanvas`, so some demos effects are not correct.

**Comment by RandomReaper** [comment]
Hi @metc, It is working fine for me, can you provide more info? platform, ...
Framerate limitation is also platform/driver dependant.

**Comment by pmudry** [comment]
Ok I'll fix this. On Linux, it seems there's no problem with that.

**Comment by metc** [comment]
@RandomReaper 
Tested on Windows 7 x64 with `java 1.8.0_40` and `java 1.6.0_45`

**Comment by pmudry** [comment]
Fixed in 58205fb1cb85891b5b67dca5524bc1bd68a4750b

**Comment by pmudry** [comment]
Linux version not working completely. Working on this

**Comment by metc** [comment]
The `DemoJuliaFractal` demo should also be fixed (Windows/Linux).

**Comment by pmudry** [comment]
Fixed in d488aced94ebc8c302803feb8279e6106afbdaa5
